frontend/src/components/energy/EnergyCookingResult.vue

```diff
@@ -873,11 +908,54 @@ export default class EnergyCookingResult extends Vue {
       costAffordability: householdResult.costAffordability * 100,
     };
   }
+
+  downloadExcel(): void {
+    this.downloadAsyncExcel("unhcr-tss-energy-household-cooking.xlsx");
+  }
+
+  async downloadAsyncExcel(filename: string): Promise<void> {
+    const workbook = new Workbook();
+    workbook.creator = "unhcr-tss.epfl.ch";
+    workbook.created = new Date();
+    zip(this.years, this.sites, this.siteResults)
+      .filter(
+        (item): item is [number, Site, SiteResult] =>
+          item[0] !== undefined &&
+          item[1] !== undefined &&
+          item[2] !== undefined
+      )
+      .forEach(([year, , siteResult]) => {
```

Comment:
@blueur isn'it better to have [year, _, siteResult] instead of [year, , siteResult] ?